docs/src/main/sphinx/release/release-465.md

```diff
@@ -0,0 +1,152 @@
+
+* Fix incorrect quoting of output values when the `CSV_UNQUOTED` or the 
+  `CSV_HEADER_UNQUOTED` format is used. ({issue}`24113`)
+
+## BigQuery connector 
+
+* Fix failure when reading views with `timestamp` columns. ({issue}`24004`)
+
+## Cassandra connector
+
+* {{breaking}} Require setting the `cassandra.security` configuration property
+  to `PASSWORD` along with `cassandra.username` and `cassandra.password` for
+  password-based authentication. ({issue}`23899`)
+
+## Clickhouse connector
+
+* Fix insert for tables with the Clickhouse `TimestampWithTimeZone` type. ({issue}`23785`)
+* Fix connector initialization issue when multiple catalogs with the connector
+  are configured. ({issue}`24058`)
+
+## Delta Lake connector
+
+* Add support for customer-provided SSE key in [S3 file
+  system](/object-storage/file-system-s3). ({issue}`22992`)
+* Fix incorrect results when filtering partition columns on checkpoints with
+  `NAME` column mapping mode. ({issue}`24104`)
+
+## Druid connector
+
+* Fix connector initialization issue when multiple catalogs with the connector
+  are configured. ({issue}`24058`)
+
+## Exasol connector
+
+* Fix connector initialization issue when multiple catalogs with the connector
+  are configured. ({issue}`24058`)
+
+## Faker connector
+
+* Add the {func}`random_string` catalog function. ({issue}`23990`)
+* Make generated data deterministic for repeated queries. ({issue}`24008`)
+* Allow configuring locale with the `faker.locale` configuration property. ({issue}`24152`)
+
+## Hive connector
+
+* Add support for skipping archiving when committing to a table in the Glue metastore
+  and the `hive.metastore.glue.skip-archive` configuration property is set to
+  `true`. ({issue}`23817`)
+* Add support for customer-provided SSE key in [S3 file
+  system](/object-storage/file-system-s3). ({issue}`22992`)
+
+## Hudi connector
+
+* Add support for customer-provided SSE key in [S3 file
+  system](/object-storage/file-system-s3). ({issue}`22992`)
+
+## Iceberg connector
+
+* Add support for reading and writing arbitrary table properties with the
+  `extra_properties` table property. ({issue}`17427`, {issue}`24031`)
+* Add the `spec_id`, `partition`, `sort_order_id`, and `readable_metrics`
+  columns to the `$files` metadata table. ({issue}`24102`)
+* Add support for configuring an OAuth2 server URI with the
+  `iceberg.rest-catalog.oauth2.server-uri` configuration property. ({issue}`23086`)
+* Add support for retrying requests to a JDBC catalog with the
+  `iceberg.jdbc-catalog.retryable-status-codes` configuration property.
+  ({issue}`23095`)
+* Add support for case-insensitive name matching in the REST catalog. ({issue}`23715`)
+* Add support for customer-provided SSE key in [S3 file
+  system](/object-storage/file-system-s3). ({issue}`22992`)
+* Improve performance of Iceberg queries involving multiple table scans. ({issue}`23945`)
+* Fix Iceberg merge, update, delete, for tables with equality deletes. ({issue}`15952`)
```

Comment:
What do you mean by "Fix" @dain .. what was going wrong before?

Reply:
If you had a table with an equality delete, so something like Flink, and then you run a Trino Merge command on that table, you get an error and the query fails.  Trino does not produce equality deltes (only position deltes), and I think the same is true for Spark, so this has to be something like Flink.  Equality deletes are thus are super rare and when you have them, the merge queries just don't work.

Reply:
Cool.. reworded a bit.